scripts/mysql_slave_lag.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+seconds=`mysql -s -e 'show slave status\G' | grep Seconds_Behind_Master | awk {'print $2'}`
```

Comment:
If Slave_IO_Running or Slave_SQL_Running is No then replication is not occurring. Since broken replication is a cause of lag maybe it should be a check called mysql_slave_status.